jupyter-R.el

```diff
@@ -11,12 +11,17 @@
         (t (cl-call-next-method))))
 
 (defun jupyter-org-result--R-html (data)
+  "Parse and convert DATA of type 'text/html' into a prettier form.
+If DATA is not a <table>, <ol>, or <dl>, returns nil."
   (with-temp-buffer
     (insert data)
     (let* ((parsed-html (libxml-parse-html-region (point-min) (point-max)))
-           (inner-element (jupyter-org-result--R-html-inner-element parsed-html)))
-      (cond ((eq (car inner-element) 'table)
-             (jupyter-org-result--R-html-table parsed-html))))))
+           (inner-element (jupyter-org-result--R-html-inner-element parsed-html))
+           (inner-element-type (car inner-element)))
+      (cond ((eq inner-element-type 'table)
```

Comment:
We can also do

```emacs-lisp
(cl-case (car inner-element)
  (table
   ...)
  ((ol dl ul)
   ...))
```

its a little more compact.

Reply:
Nice! I've done this as you suggested.